parsec/core/gui/parsec_application.py

```diff
@@ -24,6 +26,10 @@ def __init__(self):
 
     def event(self, e):
         """Handle macOS FileOpen events."""
+        if sys.platform == "darwin" and e.type() == QEvent.ApplicationActivate:
```

Comment:
According to https://doc.qt.io/qt-5/qevent.html, `QEvent.ApplicationActivate` is deprecated (and should be replaced by `ApplicationStateChange`), any idea why is it so ?

Reply:
There are three events related to this issue: `ApplicationActivate`, `ApplicationDeactivate` and `ApplicationStateChange`.

The events `ApplicationDeactivate` and `ApplicationStateChange` are tied to whenever the window state changes from foreground to background. Inversely, the events `ApplicationActivate` and `ApplicationStateChange` can be caught when switching from background to foreground, or clicking the dock icon.

Even though `ApplicationActivate` is said to be deprecated, it's seemingly the only event we can use for this particular case, `ApplicationStateChange` not being specific enough

Reply:
Great explanation ! You should add it to the code ;-)